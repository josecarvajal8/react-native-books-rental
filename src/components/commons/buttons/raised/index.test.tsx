import React from 'react';
import { Text } from 'react-native-animatable';
import renderer from 'react-test-renderer';
import RaisedButton from '.';
const mockProps = {
    children: () => <Text>{'hello'}</Text>,
    action: () => null
}
test('Render raisedButton component correctly', () => {
    const raisedButtonCmp = renderer.create(<RaisedButton {...mockProps} />).toJSON();
    expect(raisedButtonCmp).toMatchSnapshot();
});