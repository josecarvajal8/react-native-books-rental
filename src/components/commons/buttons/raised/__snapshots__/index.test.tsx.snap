// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render raisedButton component correctly 1`] = `
<RNGestureHandlerButton
  collapsable={false}
  onGestureEvent={[Function]}
  onGestureHandlerEvent={[Function]}
  onGestureHandlerStateChange={[Function]}
  onHandlerStateChange={[Function]}
  rippleColor={0}
>
  <View
    accessible={true}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#0baee6",
        "borderRadius": 30,
        "height": 45,
        "justifyContent": "center",
        "opacity": 1,
        "padding": 10,
      }
    }
  />
</RNGestureHandlerButton>
`;
